src/app/loadout-builder/generated-sets/CompareDrawer.tsx

```diff
@@ -75,6 +75,17 @@ function mapStateToProps() {
   });
 }
 
+function chooseSimilarLoadout(
+  setItems: DimItem[],
+  useableLoadouts: Loadout[]
+): Loadout | undefined {
+  const exotic = setItems.find((i) => i.isExotic);
```

Comment:
`DimItem.isExotic` is largely hangover from destiny 1. We use the following now,
```suggestion
  const exotic = setItems.find((i) => i.equippingLabel);
```